### ISSUE TITLE
How to use padding_mode="reflection" in grid_sampler

### ISSUE THREAD

**Issue by R132** [PaddlePaddle/Paddle#28245]
如题，对应pytorch中API为：
https://pytorch.org/docs/stable/nn.functional.html?highlight=grid_sample#torch.nn.functional.grid_sample

torch.nn.functional.grid_sample(input, grid, mode='bilinear', padding_mode='zeros', align_corners=None)

padding_mode="reflection": use values at locations reflected by the border for out-of-bound grid locations. For location far away from the border, it will keep being reflected until becoming in bound, e.g., (normalized) pixel location x = -3.5 reflects by border -1 and becomes x' = 1.5, then reflects by border 1 and becomes x'' = -0.5.

**Comment by cryoco** [comment]
请问您是想问，paddle中是否有对应torch中相应功能的api吗？

**Comment by R132** [comment]
> 请问您是想问，paddle中是否有对应torch中相应功能的api吗？

应该是没有吧，想咨询一下有没有遇到类似的问题，我感觉似乎想要实现需要循环不断处理，直到把值映射到 [-1, 1] 范围

**Comment by R132** [comment]
> 请问您是想问，paddle中是否有对应torch中相应功能的api吗？

有一个问题就是paddle怎么实现mask*Tensor呢？

**Comment by cryoco** [comment]
> > 请问您是想问，paddle中是否有对应torch中相应功能的api吗？
> 
> 应该是没有吧，想咨询一下有没有遇到类似的问题，我感觉似乎想要实现需要循环不断处理，直到把值映射到 [-1, 1] 范围

您好，从2.0beta版本后padding_mode这个参数我们已经支持，使用方法和torch类似，您可以参考这里的文档：https://github.com/PaddlePaddle/Paddle/blob/release/2.0-beta/python/paddle/nn/functional/vision.py#L196
官网api文档尚未更新，已经反馈相关同学，我们会尽快更新。

**Comment by R132** [comment]
> > > 请问您是想问，paddle中是否有对应torch中相应功能的api吗？
> > 
> > 
> > 应该是没有吧，想咨询一下有没有遇到类似的问题，我感觉似乎想要实现需要循环不断处理，直到把值映射到 [-1, 1] 范围
> 
> 您好，从2.0beta版本后padding_mode这个参数我们已经支持，使用方法和torch类似，您可以参考这里的文档：https://github.com/PaddlePaddle/Paddle/blob/release/2.0-beta/python/paddle/nn/functional/vision.py#L196
> 官网api文档尚未更新，已经反馈相关同学，我们会尽快更新。

刚好正在用beta，好我试试。

**Comment by R132** [comment]
> > > 请问您是想问，paddle中是否有对应torch中相应功能的api吗？
> > 
> > 
> > 应该是没有吧，想咨询一下有没有遇到类似的问题，我感觉似乎想要实现需要循环不断处理，直到把值映射到 [-1, 1] 范围
> 
> 您好，从2.0beta版本后padding_mode这个参数我们已经支持，使用方法和torch类似，您可以参考这里的文档：https://github.com/PaddlePaddle/Paddle/blob/release/2.0-beta/python/paddle/nn/functional/vision.py#L196
> 官网api文档尚未更新，已经反馈相关同学，我们会尽快更新。

windows下的paddlepaddle-gpu-2.0.0b0 还没有这个参数哈，麻烦更进一下QAQ

-----更新
发现新版本在paddle.nn.functional.grid_sample，不是之前的grid_sampler

**Comment by R132** [comment]
> > > > 请问您是想问，paddle中是否有对应torch中相应功能的api吗？
> > > 
> > > 
> > > 应该是没有吧，想咨询一下有没有遇到类似的问题，我感觉似乎想要实现需要循环不断处理，直到把值映射到 [-1, 1] 范围
> > 
> > 
> > 您好，从2.0beta版本后padding_mode这个参数我们已经支持，使用方法和torch类似，您可以参考这里的文档：https://github.com/PaddlePaddle/Paddle/blob/release/2.0-beta/python/paddle/nn/functional/vision.py#L196
> > 官网api文档尚未更新，已经反馈相关同学，我们会尽快更新。
> 
> windows下的paddlepaddle-gpu-2.0.0b0 还没有这个参数哈，麻烦更进一下QAQ

我错了，新版本改名回grid_sample 不是 grid_sampler 了，需要区分
